dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2199,13 +2202,14 @@ RtpsUdpDataLink::build_meta_submessage_map(MetaSubmessageVec& meta_submessages,
   AddrSet addrs;
   // Sort meta_submessages by address set and destination
   for (MetaSubmessageVec::iterator it = meta_submessages.begin(); it != meta_submessages.end(); ++it) {
-    if (it->dst_guid_ == GUID_UNKNOWN) {
+    const bool undirected = it->dst_guid_ == GUID_UNKNOWN;
```

Comment:
Would this be clearer if it was `directed`?

Reply:
or `prefer_multicast` or `prefer_unicast`?

Reply:
In the context of this function, I prefer directed. For RemoteInfo & accumulate_addresses I'm open to switching to different names if there's consensus.